### DISCUSSION TITLE
Powershell - additional new line after every prompt

### DISCUSSION THREAD

**Discussion by ksawerykarwacki** [starship/starship#1609] [Q&A]
#### Current Behavior
On Powershell 7 any prompt is followed by two new lines (one from powershell itself, one from starship)
![image](https://user-images.githubusercontent.com/11694791/90955134-2a4cad80-e47b-11ea-9768-1f6b76a52ee3.png)

#### Expected Behavior
Only add new lines generated by powershell itself or at least option to disable this behaviour
![image](https://user-images.githubusercontent.com/11694791/90955104-de9a0400-e47a-11ea-9970-a4158565477b.png)

#### Environment
- Starship version: 0.44.0
- powershell version: <unknown version>
- Operating system: Windows Windows 10 10.0.19042
- Terminal emulator: <unknown terminal> <unknown version>

#### Relevant Shell Configuration

```bash

Import-Module -Name DockerCompletion

# Ensure that Get-ChildItemColor is loaded
Import-Module Get-ChildItemColor

# Set l and ls alias to use the new Get-ChildItemColor cmdlets
Set-Alias l Get-ChildItemColor -Option AllScope
Set-Alias ls Get-ChildItemColorFormatWide -Option AllScope

Invoke-Expression (&starship init powershell)
```

#### Starship Configuration

```toml
<unknown config>
```

**Comment by andytom** [comment]
Hi @ksawerykarwacki , have you tried setting `add_newline` to `false` in your config file, https://starship.rs/config/#prompt?
